microsoft-edge/webview2/samples/wv2deploymentwixburnbundlesample.md

```diff
@@ -141,7 +141,7 @@ Follow the steps in that section, and then return to this page and continue belo
 
 1. This sample is an extension to the [WV2DeploymentWiXCustomActionSample](./wv2deploymentwixcustomactionsample.md) sample.  In Solution Explorer, expand **WV2DeploymentWiXCustomActionSample** and then double-click `Product.wxs`.
 
-1. In `Product.wxs`, comment out all the `<Binary>`, `<CustomAction>`, and `<Custom>` elements under `<!-- Step 4: Config Custom Action to download/install Bootstrapper -->` and `<!-- Step 5: Config execute sequence of custom action -->` so that Custom Action is not used.
+1. In `Product.wxs`, comment out all the `<Binary>`, `<CustomAction>`, and `<Custom>` elements under `<!-- Step 4: Config Custom Action to download/install Bootstrapper -->` and `<!-- Step 5: Config run sequence of custom action -->` so that Custom Action is not used.
```

Comment:
```suggestion
1. In `Product.wxs`, comment out all the `<Binary>`, `<CustomAction>`, and `<Custom>` elements under `<!-- Step 4: Config Custom Action to download/install Bootstrapper -->` and `<!-- Step 5: Config execute sequence of custom action -->` so that Custom Action is not used.
```
reverting b/c code
https://github.com/MicrosoftEdge/WebView2Samples/blob/main/SampleApps/WV2DeploymentWiXCustomActionSample/Product.wxs - line 64 & 65; 65 is `<InstallExecuteSequence>`

An xml comment in the code repo is out of scope; would need to create a PR in that code repo before (or simult. w/) changing here in docs repo.  Such a PR might get rejected, since the xml code itself uses the word/phrase 'ExecuteSequence': line 65 is `<InstallExecuteSequence>`.

Reply:
approved comment